src/components/App/App.js

```diff
@@ -1,121 +1,102 @@
-import React, { Component } from 'react';
-import { people } from '../../constants/people';
-import { ButtonAsClass as Button } from '../Button/Button';
-import { Header } from '../Header/Header';
-import Checkbox from '../Checkbox/Checkbox';
-import { Input } from '../Input/Input';
+import React, {Component} from 'react';
+import {Button as Button} from '../Button/Button';
+import {Header} from '../Header/Header';
+import {Form} from '../Form/Form';
+import {Toggle} from '../Toggle/Toggle'
+import Switch from 'react-switch';
```

Comment:
Nice, but can you please also create your own toggle that would work in same way as this Switch?